.github/workflows/trunk.yml

```diff
@@ -256,3 +256,7 @@ jobs:
     secrets:
       AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID: ${{ secrets.AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID }}
       AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY: ${{ secrets.AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY }}
+
+  linux-ubuntu-android-tests:
```

Comment:
Let's rename to "android-emulator-build-test". Linux or Ubuntu is not really relevant here.